src/libraries/packages.builds

```diff
@@ -2,7 +2,7 @@
   <Import Project="$(MSBuildThisFileDirectory)Directory.Build.props" />
 
   <PropertyGroup Condition="'$(DotNetBuildFromSource)' == 'true'">
-    <AdditionalBuildConfigurations>$(AdditionalBuildConfigurations);package-$(Configuration)</AdditionalBuildConfigurations>
+    <AdditionalBuildTargetFrameworks>$(AdditionalAdditionalBuildTargetFrameworks);package-$(Configuration)</AdditionalBuildTargetFrameworks>
```

Comment:
This doesn't look right:

1) Typo
2) Was $(Configuration) here refereing to the the expanded string or really just to the configuration string?

cc @ericstj

Reply:
It was just the Debug or Release